justin-tan/week5/tues/rails-crud-hw/app/views/restaurants/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2>Restaurant Detail</h2>
+
+<img src="<%= @restaurant.restaurant_img %>" alt="<%= @restaurant.name %>">
+
+<h3>Name</h3>
+<p><%= @restaurant.name %></p>
+
+<h3>Chef</h3>
+<p><%= @restaurant.chef %></p>
+
+<h3>Location</h3>
+<p><%= @restaurant.location %></p>
+
+<h3>Cuisine</h3>
+<p><%= @restaurant.cuisine %></p>
+
+<h3>Year</h3>
+<p><%= @restaurant.year %></p>
+
+<h3>Price Range</h3>
+<p>$<%= @restaurant.price_range %> pp.</p>
+
+<h3>Description</h3>
+<p><%= @restaurant.description %></p>
+
+<h3>Notable Dish</h3>
+<img src="<%= @restaurant.dish_img %>" alt="<%= @restaurant.dish %>" id="show_img">
```

Comment:
Let rails help you with their image tag
use `<%= image_tag @restaurant.dish_img, alt: @restaurant.dish, id: "show_img"  %>` instead